### PR TITLE
Monitor: support monitoring lightning with TLS enabled

### PR DESCRIPTION
**Pull request by shuijing198799**
### What problem does this PR solve? 
When the monitor tls enable, the scheme of lightning in prom rules should be https instead of http 

Tests 

 - Unit test
 - E2E test

Code changes

 - Has Go code change

### CODE REVIEW

File: pkg/monitor/monitor/template.go [hunk 1]
```diff
@@ -381,7 +381,7 @@ func addTlsConfig(pc *config.Config) {
 		// TODO support tiflash tls when it gets ready
 		if sconfig.JobName == "pd" || sconfig.JobName == "tidb" || sconfig.JobName == "tikv" ||
 			sconfig.JobName == "pump" || sconfig.JobName == "drainer" ||
-			sconfig.JobName == "tiflash" || sconfig.JobName == "tiflash-proxy" {
+			sconfig.JobName == "tiflash" || sconfig.JobName == "tiflash-proxy" || sconfig.JobName == "lightning" {
```
Comment (DanielZhangQD) [inline pkg/monitor/monitor/template.go:384]: The lightning chart does not support TLS yet.

Comment (shuijing198799) [reply pkg/monitor/monitor/template.go:384]: DBaaS will not use lightning chart, Just set scheme is all right.

**Review by LinuxGit — APPROVED** [review]
LGTM

File: pkg/monitor/monitor/template.go [hunk 2]
```diff
@@ -390,6 +390,11 @@ func addTlsConfig(pc *config.Config) {
 			pc.ScrapeConfigs[id] = sconfig
 			sconfig.Scheme = "https"
 		}
+		// lightning does not need to monitor the access of other components,
```
Comment (DanielZhangQD) [inline pkg/monitor/monitor/template.go:393]:
```suggestion
		// lightning does not need to verify the access of other components,
```

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Pull request merged**